src/multibanco/multibanco.js

```diff
@@ -0,0 +1,30 @@
+// 0. Get originKey
+getOriginKey().then(originKey => {
+    getPaymentMethods().then(paymentMethodsResponse => {
+        // 1. Create an instance of AdyenCheckout
+        const checkout = new AdyenCheckout({
+            originKey, // Mandatory. originKey from Costumer Area
+            environment: 'test',
+            amount: { currency: 'EUR', value: 1000 },
+            onAdditionalDetails: result => {
+                console.log(result);
+            },
+            onError: error => {
+                console.log(error);
+            },
+            onSubmit: initialSubmit,
+            paymentMethodsResponse
+        });
+
+        function initialSubmit (state, component) {
+            makePayment(state.data).then(response => {
+                initialComponent.unmount();
+                // 3. present the voucher using the action object returned from /payments
+                checkout.createFromAction(response.action).mount('#multibanco-container');
+            });
+        }
+
+        // 2. create Multibanco component
+        const initialComponent = checkout.create('multibanco').mount('#multibanco-container');
```

Comment:
```suggestion
        checkout.create('multibanco').mount('#multibanco-container');
```

As the `onSubmit` event will already have a reference to the current component we can maybe remove this variable.

Reply:
Of course! It's passed in explicitly. Will rectify :)